src/services/urlAssembly.spec.ts

```diff
@@ -150,6 +196,22 @@ describe('query params', () => {
     expect(url).toBe('/?simple=')
   })
 
+
+  test('given route with default string param provided but empty, returns route Query with string without values interpolated', () => {
+    const [route] = createRoutes([
+      {
+        name: 'simple',
+        path: '/',
+        query: query('simple=[simple]', { simple: withDefault(String, 'abc') }),
+        component,
+      },
+    ])
+
+    const url = assembleUrl(route, { params: { simple: '' } })
+
+    expect(url).toBe('/?simple=')
+  })
```

Comment:
Same as the path test. This is the same even if there is no default right? Probably don't need this test do we?